docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3.0'
 services:
   searcharr:
     container_name: searcharr
-    image: toddrob/searcharr:latest
+    image: toddrob/searcharr:beta
```

Comment:
This is the default docker-compose.yml for new users, so it should point to `latest`.

Reply:
ok